src/routes/recovery/recovery.controller.ts

```diff
@@ -26,4 +35,17 @@ export class RecoveryController {
       addRecoveryModuleDto,
     });
   }
+
+  @HttpCode(200)
```

Comment:
If we don't return a body in the response, then `204 No Content` might be a better fit.

Reply:
Changed in 82952c4763b1f9a0705743980b8febc963eafa67.